pyigm/guis/igmguesses.py

```diff
@@ -1003,6 +1015,11 @@ def on_key(self,event):
         if event.key == 'P':  # Toggle colorful mode
             self.flag_colorful = not self.flag_colorful
 
+        # print component info
+        if event.key == '@':  #  for develop; this will be deleted in future.
```

Comment:
I'm unclear.  Is this to be used?  If not, why is it here?

Other than this issue, all looks fine.  Merge when ready